front/lib/api/assistant/email_trigger.ts

```diff
@@ -309,46 +309,39 @@ export async function triggerFromEmail({
   // console.log("REST_OF_THREAD", restOfThread, restOfThread.length);
 
   if (restOfThread.length > 0) {
-    const file = await FileResource.makeNew({
-      contentType: "text/plain",
+    const cfRes = await toFileContentFragment(auth, {
+      contentFragment: {
+        title: `Email thread: ${email.subject}`,
+        content: restOfThread,
+        contentType: "text/plain",
+        url: null,
+      },
       fileName: `email-${email.subject}.txt`,
```

Comment:
nit: the filename might be truncated to 256 characters.

Reply:
will call it email-thread.txt